### PR TITLE
Update language Italian.0.9.0part1.js

### PR DESCRIPTION
**Pull request by francescaneri**
begun to translate profile editor. Matteo

### CODE REVIEW

File: lib/language.js [hunk 1]
```diff
@@ -1366,7 +1382,20 @@ function init() {
...
+      it: 'NS - Dati Personali'
+      }
+    ,'General Profile setting' : {
+      it: 'Impostazioni Generali'
+      }
+    ,'Title' : {
+      it: 'Titolo'
+      } 
+    ,'Units' : {
+      ,it: 'Unità'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241525904/issues/source?bid=2452379&fileBranchId=2935957#l1397)

Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241525904/issues/source?bid=2452379&fileBranchId=2935957#l1397)

**Comment by francescaneri** [comment]
@jasoncalabrese what other files I have to look to see the connection ?

File: lib/language.js [hunk 2]
```diff
@@ -794,6 +794,22 @@ function init() {
       ,nb: '<ingen>'
       ,he: '<ללא>'
       }
+      'none' : {
+      cs: 'Žádný'
+      ,de: 'nichts'
+      ,es: 'ninguno'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241765613/issues/source?bid=2452379&fileBranchId=2935957#l800)

File: lib/language.js [hunk 3]
```diff
@@ -794,6 +794,22 @@ function init() {
       ,nb: '<ingen>'
       ,he: '<ללא>'
       }
+      'none' : {
+      cs: 'Žádný'
+      ,de: 'nichts'
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241765613/issues/source?bid=2452379&fileBranchId=2935957#l799)

File: lib/language.js (hunk 2 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241765613/issues/source?bid=2452379&fileBranchId=2935957#l800)

File: lib/language.js (hunk 3 above)
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241765613/issues/source?bid=2452379&fileBranchId=2935957#l799)

File: lib/language.js [hunk 4]
```diff
@@ -794,6 +794,22 @@ function init() {
       ,nb: '<ingen>'
       ,he: '<ללא>'
       }
+      'none' : {
```
Comment (jasoncalabrese) [inline lib/language.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241765613/issues/source?bid=2452379&fileBranchId=2935957#l797)

**Comment by jasoncalabrese** [comment]
We're in the middle of trying get `dev-0.9` merged into `dev` think that was part of the issue with the merge conflicts before.  Might be good to wait till we get this merge in place.  The 0.9 release is going to be weeks or maybe a month or more away, so there's no rush.

**Pull request closed without merging**